### PR TITLE
sports.yahoo.co.jp/m/keiba/ 

### PR DESCRIPTION
**Pull request by SKEIDs**
https://sports.yahoo.co.jp/m/keiba/

```
sports.yahoo.co.jp##aside[id^="w2aLink_"]
sports.yahoo.co.jp###prm_sapp
```

<details><summary>screenshot</summary>

</details><br/>

### CODE REVIEW

File: AnnoyancesFilter/MobileApp/sections/mobile-app_specific.txt [hunk 1]
```diff
@@ -1016,6 +1016,8 @@ weather.yahoo.co.jp##a[href^="https://app.adjust.com/"]
 shopping.yahoo.co.jp,103.by##.AppBanner
 search.yahoo.co.jp###contentsInner > article[class^="W2ALink_W2ALink__"]
 sports.yahoo.co.jp###header > #RSAB
+sports.yahoo.co.jp##aside[id^="w2aLink_"]
```
Comment (Yuki2718) [inline AnnoyancesFilter/MobileApp/sections/mobile-app_specific.txt:1019]: Maybe better `sports.yahoo.co.jp###prm_text`

**Review by Yuki2718 — COMMENTED** [review]
I don't see `###prm_sapp`, can you specify in screenshot? OTOH
```
yahoo.co.jp###RSAB
yahoo.co.jp###prm_apbn
```
need `!+ NOT_OPTIMIZED`.

**Comment by SKEIDs** [comment]
> I don't see `###prm_sapp`, can you specify in screenshot? 

`###prm_sapp` is "おすすめアプリ".

<details><summary>screenshot</summary>

[](https://postimg.cc/xJv0pRKr)
</details><br/>

File: AnnoyancesFilter/MobileApp/sections/mobile-app_specific.txt (hunk 1 above)
Comment (SKEIDs) [reply AnnoyancesFilter/MobileApp/sections/mobile-app_specific.txt:1019]: ok, I will fix

**Comment by Yuki2718** [comment]
> `###prm_sapp` is "おすすめアプリ".

Okay, apparently they show that section only for Chrome/Safari and not for Firefox.

> Can this therefore be do after merged this?

I'll directly edit.

**Review by Yuki2718 — APPROVED** [review]

**Review by zloyden — APPROVED** [review]

**Pull request merged**